package.json

```diff
@@ -53,6 +50,7 @@
   "engines": {
     "node": ">=4"
   },
+  "files":["@hoodie/admin", "@hoodie/client", "@hoodie/server"],
```

Comment:
e.g. see https://github.com/hoodiehq/hoodie-admin/blob/master/package.json#L14-L16. This means that when the package get published to the npm registry, only the contents of the `dist/` folder get uploaded, everything else is ignored. Does that  make sense